winterfell/webapp/src/js/formRenderingService.js

```diff
@@ -27,3 +27,29 @@ app.factory('formRenderingService', ['$window', '$http', function ($window, $htt
         }
     }
 }]);
+
+/**
+ * Service to retrieve the formly template (which is just a static JSON file)
+ * for a given form.
+ */
+app.factory('formTemplateService', ['$http', function ($http) {
```

Comment:
it's not in package.json. It's a custom file on the same level as this file 'webapp/src/js/net.js'.